python/tank/bootstrap/manager.py

```diff
@@ -215,10 +202,10 @@ def _get_progress_callback(self):
 
             progress_callback(progress_value, message)
 
-        where:
-        - ``progress_value`` is the current progress value, a float number ranging from 0.0 to 1.0
-                             representing the percentage of work completed.
-        - ``message`` is the progress message string to report.
+            # progress_value is the current progress value, a float number
+            # ranging from 0.0 to 1.0 and representing the percentage of work completed.
+            #
+            # message is the progress message string to report.
```

Comment:
...and the other place:

![image](https://cloud.githubusercontent.com/assets/337710/18587094/101689f6-7c18-11e6-833a-788ced477ed0.png)